tests/e2e/specs/in-game/game-over/winConditions.spec.js

```diff
@@ -385,38 +374,120 @@ describe('Stalemates', () => {
         .get('[data-cy=request-gameover-confirm]')
         .click();
       cy.get('#waiting-for-opponent-stalemate-scrim').should('be.visible');
+      cy.reload();
+      cy.get('#waiting-for-opponent-stalemate-scrim').should('be.visible');
+      cy.acceptStalemateOpponent();
+      assertStalemate();
+    });
 
-      cy.drawCardOpponent();
-      cy.get('#waiting-for-opponent-stalemate-scrim').should('not.exist');
+    it('Rejects a stalemate while resolving a seven', () => {
```

Comment:
This confirms we can request + reject a stalemate while in the middle of a different phase and then get back to that phase to accept the next real move after its rejected (the payoff of adding `priorStates` to all the move helpers)